src/tests/golden/testdata/quickfix/getTypeMissUse.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+function getTypeMissUse(mixed $var){
+    if (gettype($var) === "string"){
+    }
+
+    if (gettype($var) == "double"){
+    }
+
+    if (gettype($var) !== "array"){
+    }
+
+    if (gettype($var) != "boolean"){
+    }
+
+    if (gettype($var) === "object"){
```

Comment:
Please, add cases for expression, not just variables

Reply:
done